source/config.js

```diff
@@ -11,6 +11,10 @@ var defaultConfig = {
 
   // The base URL ungit will be accessible from.
   urlBase: "http://localhost",
+  
+  // The URL root path under which ungit will be accesible.
+  // Either blank or starting with a slash and no trailing slash
```

Comment:
Perhaps we can use [url.resolve()](https://nodejs.org/api/url.html#url_url_resolve_from_to)?  Not sure if we need additional validation though...